### ISSUE TITLE
Offer minified css from Gravatar

### ISSUE THREAD

**Issue by bhubbard** [Automattic/jetpack#4251]
Jetpack loads two css files from Gravatar. 

```
https://secure.gravatar.com/css/services.css
https://secure.gravatar.com/css/hovercard.css
```

* Can these be combined into one css file? 
* Can we load/offer minified versions of these files?

**Comment by amityweb** [comment]
Google Page Speed Insights score is lowered due to https://secure.gravatar.com/css/hovercard.css not being minified. Solution is to uncheck use Avatar and hovercards in Worpdress Admin until the CSS follows Google best practice.

**Comment by bhubbard** [comment]
I still would like to see this get updated.

**Comment by kraftbj** [comment]
I've cross-posted this to the Gravatar's team tracker.

**Comment by bhubbard** [comment]
@kraftbj any update on this?

**Comment by kraftbj** [comment]
No, the team maintaining Gravatar has it on their list, but no update at this time.
